pkg/util/containers/image.go

```diff
@@ -10,6 +10,13 @@ import (
 	"strings"
 )
 
+var (
+	// ErrEmptyImage is returned when argument is empty
```

Comment:
Nit - image name is empty?
```suggestion
	// ErrEmptyImage is returned when image name argument is empty
```